src/git/utils.ts

```diff
@@ -218,7 +218,7 @@ async function listObjects({
         }
         return;
       default:
-        utils.never();
+        utils.never('Invalid type');
```

Comment:
Include what the type was that is invalid. Also include what the allowed types are in the message.